.github/workflows/release.yml

```diff
@@ -0,0 +1,61 @@
+name: Release
+
+on:
+  push:
+    # Sequence of patterns matched against refs/tags
+    tags:
+      # TODO: Switch to tighter version-number matching.
+      - '[0-9]+.*' # Push events to tags starting with numbers
+
+jobs:
+  build:
+    name: Create Release
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+      # Set up Node
+      - name: Use Node.js 12
+        uses: actions/setup-node@v1
+        with:
+          node-version: 12.x
+          registry-url: 'https://registry.npmjs.org'
+      # Install dependencies
+      - name: Install dependencies
+        run: yarn install --frozen-lockfile
+      - name: Build project
+        # I couldn't quickly figure out how to get just the tag name in both a `run` context and as an input to the release asset action, below.
+        # See https://github.community/t5/GitHub-Actions/How-to-get-just-the-tag-name/td-p/31595
```

Comment:
Yeah, exactly. I'd like to have the zip file be named `loam-1.0.0.zip` or whatever. I got it working under a `run` context using the linked thread, but as an input to a task it doesn't seem like there's access to shell interpolation, so the same approach didn't work there. 😞

Reply:
So actually your link to the changelog-parsing example above helped me to figure this out, even though it's no longer necessary now that I've opted not to generate a custom release asset.

However, in case we run into this again, here's an explanation: The thread [here](https://github.community/t5/GitHub-Actions/How-to-get-just-the-tag-name/m-p/32167#M1027) has the full solution. It's using shell parsing in a `run` command to slice the string appropriately (which I understood the first time through), and then denoting that as an output of the step, so that other steps can access the result (which I didn't). Denoting it as a step output allows other steps to access the value via `${{ steps.get_version.outputs.VERSION }}`.

Reply:
Oh cool, I didn't know about `::set-output`! That seems like an excellent solution moving forward.